controller/token.go

```diff
@@ -183,7 +184,7 @@ func (c *TokenController) getKeycloakIdentityProviderURL(identityID string, prov
 func (c *TokenController) Retrieve(ctx *app.RetrieveTokenContext) error {
 	appToken, errorResponse, err := c.retrieveToken(ctx, ctx.For, ctx.RequestData, ctx.ForcePull)
 	if errorResponse != nil {
-		ctx.ResponseData.Header().Set("Access-Control-Expose-Headers", "WWW-Authenticate")
+		ctx.ResponseData.Header().Add("Access-Control-Expose-Headers", "WWW-Authenticate")
```

Comment:
Why are we setting instead of adding? I feel it 'safe' to not overwrite, and only add as needed :)

Reply:
This is exactly why I changed from setting to adding ;)

Reply:
Oh right :| What was I seeing.